modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValidatorImpl.java

```diff
@@ -322,6 +324,33 @@ private void _validateDDMFormFieldPropertyValue(
 		}
 	}
 
+	private void _validateDDMFormFieldReferences(
+			List<DDMFormField> ddmFormFields)
+		throws DDMFormValidationException {
+
+		if (ddmFormFields.isEmpty()) {
+			return;
+		}
```

Comment:
@devpedrotavares we are already checking it [here](https://github.com/liferay-forms/liferay-portal/pull/2052/files#diff-c312a5bafbece2a3f3bf36526bca133aceb5ab11d1288f52d143134ce74182ffR89-R90)

Reply:
Okay! I removed the unnecessary check.